### PR TITLE
LBSD-88 Show original creator in blog settings

### PR DESCRIPTION
**Pull request by vladnicoara**
Show image, full name and username

### CODE REVIEW

**Comment by nistormihai** [comment]
LGTM

File: client/app/scripts/liveblog-edit/module.js [hunk 1]
```diff
@@ -194,8 +194,8 @@ define([
         });
     }
 
-    BlogSettingsController.$inject = ['$scope', 'blog', 'api', 'blogService', '$location', 'notify'];
-    function BlogSettingsController($scope, blog, api, blogService, $location, notify) {
+    BlogSettingsController.$inject = ['$scope', 'blog', 'api', 'blogService', '$location', 'notify', 'gettext'];
+    function BlogSettingsController($scope, blog, api, blogService, $location, notify, gettext) {
```
Comment (vied12) [inline client/app/scripts/liveblog-edit/module.js]: Why importing `gettext` here ?

File: client/spec/blog_settings_spec.js [hunk 2]
```diff
@@ -60,5 +60,24 @@ describe('Blog settings', function() {
...
+        openBlog(0);
+        openSettings();
+        element(by.css('[data="blog-settings-users"]')).click();
+        //expect original creator name
+        element(by.css('[data="original-creator-display-name"]')).getText().then(function(text) {
+            expect(text).toEqual('first name last name');
+        });
+    });
+
+    it('shows original creator username', function() {
```
Comment (vied12) [inline client/spec/blog_settings_spec.js]: It's maybe better to merge these two tests in the same in order to speed up the e2e tests

File: client/app/scripts/liveblog-edit/module.js (hunk 1 above)
Comment (vladnicoara) [inline client/app/scripts/liveblog-edit/module.js]: it was used a bit later with notify.info(gettext('saving blog settings')); and travis was showing an error from time to time.... something like gettext module is not imported

File: client/spec/blog_settings_spec.js (hunk 2 above)
Comment (vladnicoara) [inline client/spec/blog_settings_spec.js]: I thought of that as well but I separated them for ease of visibility when one of the expectations are not met

**Pull request merged**